src/state/api.js

```diff
@@ -54,3 +54,32 @@ export async function getProbeData(params, token) {
   });
   return data;
 }
+
+export async function getSearchResults(queryString, resultsLimit) {
+  const getFormattedSearchURL = (str, product = 'desktop') => {
+    const URLResult = new URL('__BASE_SEARCH_DOMAIN__');
+    const params = new URLSearchParams();
+    const queryOptions = [];
+
+    queryOptions.push(`name.eq.${str}`);
+    queryOptions.push(`index.phfts(english).${str}`);
+    queryOptions.push(`description.phfts(english).${str}`);
+    queryOptions.push(`name.ilike.*${str}*`);
+
+    params.set('limit', resultsLimit);
+    params.set('select', 'name,definition,type');
+    params.set('product', `eq.${product}`);
+    params.set('type', 'neq.event');
```

Comment:
The `type` may depend on product. Although I don't think this hurts fenix, or other glean products currently.

Reply:
Yep I expect adjustments here - should be easier now that it's not a long string.

Reply:
I think we'll probably understand more when we start pointing this at Fenix probes in GLAM, but let's expect these to be moved to `firefox-desktop.js` at some point